common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2260,6 +2261,14 @@ object CelebornConf extends Logging {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("4MB")
 
+  val HA_MASTER_RATIS_LOG_WRITE_BUFFER_SIZE: ConfigEntry[Long] =
```

Comment:
Why `ratis.raft.server.log.write.buffer.size` need 64MB in Celeborn while the default size is 8MB?

Reply:
@RexXiong, sorry for mistake that the default value of `ratis.raft.server.log.write.buffer.size` is `8MB` not `64MB`.